spec/system/drivers/editing_incidents_spec.rb

```diff
@@ -7,18 +7,49 @@
   before(:each) { when_current_user_is driver }
   let(:report) { create :incident_report, :driver_report, user: driver }
   let!(:incident) { create :incident, driver_incident_report: report }
+  before :each do
+    visit incidents_url
+    expect(page).to have_selector 'table.incidents tbody tr', count: 1
+    within 'tr', text: driver.proper_name do
+      click_button 'Edit'
+    end
+  end
   context 'admin deletes the incident' do
     it 'displays a nice error message' do
-      visit incidents_url
-      expect(page).to have_selector 'table.incidents tbody tr', count: 1
-      within 'tr', text: driver.proper_name do
-        click_button 'Edit'
-      end
       expect(page).to have_content 'Editing Driver Account of Incident'
       incident.destroy
       click_button 'Save report'
       wait_for_ajax!
-      expect(page).to have_selector 'p.notice', text: 'This incident report no longer exists.'
+      expect(page).to have_selector 'p.notice',
+                                    text: 'This incident report no longer exists.'
+    end
+  end
+  context 'adding multiple injured passengers' do
```

Comment:
Actually, before I write any tests for witnesses. Should this test be in `spec/system/drivers/special_incident_fields_spec.rb`?

Reply:
I don't remember why I made that file, but it's a bad filename (note the lack of a gerund helpfully describing the use case under test). What's in there?

Reply:
It tests which fields are necessary/required